src/item/Item.php

```diff
@@ -282,6 +284,23 @@ public function clearNamedTag() : Item{
 		return $this;
 	}
 
+	/**
+	 * Returns the repair cost of the item.
+	 */
+	public function getRepairCost() : int{
+		return $this->repairCost;
+	}
+
+	/**
+	 * Sets the repair cost of the item.
+	 *
+	 * @return $this
+	 */
+	public function setRepairCost(int $cost) : self{
+		$this->repairCost = $cost;
+		return $this;
+	}
+
```

Comment:
This should be placed on Durable class

Reply:
Any item can have repairCost, e.g EnchantedBook

Reply:
How do you repair an enchanted book ?

Reply:
> How do you repair an enchanted book ?

it's more like “anvil cost” rather than actual repair cost since it also increases when combining enchantments / renaming.

Reply:
So should we move away from vanilla and go in the direction of AnvilCost and not RepairCost in the item tags? I personally don't think so.

Reply:
Consistency within PM is more important, right? I think we can use anvil cost terminology and just document that in vanilla it is called repair cost.

Reply:
Yeah but the tag need to stay at RepairCost

Reply:
Okay, so document and/or name this function properly, when I saw it I thought it was the experience cost of the item to be repaired.